WordPress/Classes/ViewRelated/Reader/ReaderTopicsCardCell.swift

```diff
@@ -2,36 +2,126 @@ import UIKit
 
 /// A cell that displays topics the user might like
 ///
-class ReaderTopicsCardCell: ReaderTopicsTableCardCell {
-    private let cellIdentifier = "TopicCell"
+class ReaderTopicsCardCell: UITableViewCell, NibLoadable {
+    // Views
+    @IBOutlet weak var containerView: UIView!
+    @IBOutlet weak var headerLabel: UILabel!
+    @IBOutlet weak var collectionView: UICollectionView!
 
-    override func configure(_ data: [ReaderAbstractTopic]) {
-        super.configure(data)
+    private(set) var data: [ReaderAbstractTopic] = [] {
+        didSet {
+            guard oldValue != data else {
+                return
+            }
 
-        headerTitle = Constants.title
+            collectionView.reloadData()
+        }
+    }
+
+    weak var delegate: ReaderTopicsTableCardCellDelegate?
+
+    func configure(_ data: [ReaderAbstractTopic]) {
+        self.data = data
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        applyStyles()
+
+        // Configure header
+        headerLabel.text = Constants.title
+
+        // Configure collection view
+        collectionView.showsHorizontalScrollIndicator = false
+        collectionView.register(ReaderInterestsCollectionViewCell.defaultNib,
+                                forCellWithReuseIdentifier: Constants.reuseIdentifier)
```

Comment:
```suggestion
                                forCellWithReuseIdentifier: ReaderInterestsCollectionViewCell.defaultReuseID)
```

Reply:
Done in: 616998ecec24851a03b947bb70a219a03766ce49